src/components/sidebar/Sidebar/index.tsx

```diff
@@ -14,54 +20,82 @@ import { trackEvent, OVERVIEW_EVENTS } from '@/services/analytics'
 import KeyholeIcon from '@/components/common/icons/KeyholeIcon'
 import OwnedSafes from '../OwnedSafes'
 
-const Sidebar = (): ReactElement => {
+type SidebarProps = {
+  hideSidebar: boolean
```

Comment:
`isHidden`?

Reply:
yep! Maybe something like `isSidebarHidden` ?

Reply:
The whole component is called sidebar, why prefix each prop with the same word?